docs/spec/light/overview.md

```diff
@@ -0,0 +1,55 @@
+# Overview
```

Comment:
isnt this just a copy of what's already in the readme?

Reply:
I've removed the overview.md